packages/broker/src/plugins/operator/VoteOnSuspectNodeHelper.ts

```diff
@@ -1,32 +1,90 @@
 import { Contract } from '@ethersproject/contracts'
-import { Wallet } from '@ethersproject/wallet'
 import type { Operator } from '@streamr/network-contracts'
 import { operatorABI } from '@streamr/network-contracts'
 import { Logger } from '@streamr/utils'
 import { OperatorServiceConfig } from './OperatorPlugin'
+import { ensureValidStreamPartitionIndex } from '@streamr/protocol'
 
 export const VOTE_KICK = '0x0000000000000000000000000000000000000000000000000000000000000001'
 export const VOTE_NO_KICK = '0x0000000000000000000000000000000000000000000000000000000000000000'
 
 const logger = new Logger(module)
 
+export class ParseError extends Error {
+    public readonly reasonText: string
+
+    constructor(reasonText: string) {
+        super(`failed to parse metadata: ${reasonText}`)
```

Comment:
We seem to have a convention that error messages start with a capital letter? (at least the user-visible error messages in `broker`/`client`)

```suggestion
        super(`Failed to parse metadata: ${reasonText}`)
```